test/irmin-pack/test_pack.ml

```diff
@@ -262,7 +262,10 @@ module Pack = struct
     t.clone_index_pack ~readonly:true >>= fun (i, r) ->
     let test w =
       let adds l =
-        List.iter (fun (k, v) -> Pack.unsafe_append ~ensure_unique:true w k v) l
+        List.iter
+          (fun (k, v) ->
+            Pack.unsafe_append ~ensure_unique:true ~overcommit:false w k v)
```

Comment:
If we're going with an optional argument, then maybe they're not necessary in these tests (plus the semantics should not change)